content/posts/newsletter/2019-03-31.md

```diff
@@ -0,0 +1,46 @@
+---
+title: The XMPP Newsletter, 31st March 2019
+save_as: 2019/03/newsletter-31-march/index.html
+url: 2019/03/newsletter-31-march/
+date: 2019-02-28 00:00
```

Comment:
This should be today's date. Same for the title and URL. You'll have to `git move` the file as well I think.